cognition_objects/conversation.py

```diff
@@ -0,0 +1,126 @@
+from typing import Dict, List, Optional, Tuple, Any
+
+from ..cognition_objects import message
+from ..business_objects import general
+from ..session import session
+from ..models import CognitionConversation
+from sqlalchemy import func, alias, Integer
+
+
+def get(conversation_id: str) -> CognitionConversation:
+    return (
+        session.query(CognitionConversation)
+        .filter(CognitionConversation.id == conversation_id)
+        .first()
+    )
+
+
+def get_all_paginated_by_project_id(
+    project_id: str, page: int, limit: int
+) -> Tuple[int, int, List[CognitionConversation]]:
+    total_count = (
+        session.query(func.count(CognitionConversation.id))
+        .filter(CognitionConversation.project_id == project_id)
+        .scalar()
+    )
```

Comment:
```suggestion
    total_count = (
        session.query(CognitionConversation.id)
        .filter(CognitionConversation.project_id == project_id)
        .count()
    )
```
I think this version improves readability

Reply:
chagned